pkg/gohai/gohai_test.go

```diff
@@ -46,7 +46,7 @@ func TestGetPayloadContainerizedWithDocker0(t *testing.T) {
 	docker0Detected = true
 	defer func() { docker0Detected = oldDocker0Detected }()
 
-	gohai := GetPayload(false)
+	gohai := GetPayload("hostname", false, false)
```

Comment:
Can we add a unit test when the second elem it true ?

Reply:
Done in [address review](https://github.com/DataDog/datadog-agent/pull/35145/commits/9445dd6ad2f9317c46b0b5f134d517ca9ccb19a4)